packages/dev/core/src/Meshes/Compression/dracoCompression.ts

```diff
@@ -123,344 +53,36 @@ export interface IDracoCompressionOptions {
  *
  * @see https://playground.babylonjs.com/#DMZIBD#0
  */
-export class DracoCompression implements IDisposable {
-    private _workerPoolPromise?: Promise<AutoReleaseWorkerPool>;
-    private _decoderModulePromise?: Promise<{ module: DecoderModule }>;
-
+export class DracoCompression extends DracoDecoder {

```

Comment:
I assume that in this case this class will not include the encoder as well?

Reply:
We will not be adding the encoder to this class since it will be deprecated. When I wrote this class originally, tree shaking wasn't a consideration. Now that it is, we have to separate these two concepts or else the decoder will pull the encoder and vice versa.

Reply:
Right now, no. I asked around about separating vs. keeping them together, and got conflicting answers, so I went with separating them which felt the safest. 

The plan so far is to deprecate DracoCompression once I've finished DracoEncoder and verified the DracoCodec API is what we want.

We _could_ add the encoder to DracoCompression later, if we wanted to make the class consistent? One idea I've thought about is eventually making DracoCompression be a wrapper for instances of both an encoder and decoder.